modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcComplexQuerySelfTest.java

```diff
@@ -209,21 +208,25 @@ public void testWrongArgumentType() throws Exception {
         }
 
         // Check non-indexed field.
-        assertThrows(SQLException.class, () -> {
-            try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Org where name = 2")) {
-                assertFalse(rs.next());
-            }
-        });
+        JdbcTestUtils.assertThrowsSqlException(
+                "For input string: \"B\"",
+                () -> {
+                    try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Org where name = 2")) {
+                        assertFalse(rs.next());
+                    }
+                });
 
         // Check indexed field.
         try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Person where name = '2'")) {
             assertFalse(rs.next());
         }
 
-        assertThrows(SQLException.class, () -> {
-            try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Person where name = 2")) {
-                assertFalse(rs.next());
-            }
-        });
+        JdbcTestUtils.assertThrowsSqlException(
+                "For input string: \"Mike Green\"",
+                () -> {
+                    try (ResultSet rs = stmt.executeQuery("select * from PUBLIC.Person where name = 2")) {
+                        assertFalse(rs.next());
+                    }
+                });
```

Comment:
```suggestion
                () -> stmt.executeQuery("select * from PUBLIC.Person where name = 2")
        );
```

Reply:
don't agree, I want to close ResultSet. Unfortunately we don't follow the rules everywhere, but it's good to close RS